apps/lwipserver2/lwipserver2_x86_64.camkes

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright 2020, Data61, CSIRO (ABN 41 687 119 230)
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+import <std_connector.camkes>;
+import <global-connectors.camkes>;
+import <TimeServer/TimeServer.camkes>;
+import <SerialServer/SerialServer.camkes>;
+
+#include <camkes-lwip-ethernet-async.h>
+#include <camkes-lwip-base.h>
+#include <camkes-fdt-bind-driver.h>
+#include <camkes-single-threaded.h>
+#include <camkes-dynamic-untyped-allocators.h>
+#include <camkes-x86-iospace-dma.h>
+#include <camkes-BenchUtiliz.h>
+#include <camkes-SerialServer-camkes-putchar-client.h>
+
+#include <echo/tuning_params.h>
+
+component LWIPServer {
+    include "lwipserver.h";
```

Comment:
What is the purpose of this include?  It causes a lot of compiler errors and removing it seems to fix them.